plugin/trino-pinot/pom.xml

```diff
@@ -500,6 +500,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <scope>runtime</scope>
```

Comment:
I assume this change is related, but I'm not sure how.

Reply:
This dependency is now transitively used.